### DISCUSSION TITLE
Attempt at syncing in 2.5.0 gets stuck on "Fetching items..."

### DISCUSSION THREAD

**Discussion by Langtude-Larson** [abraunegg/onedrive#2828] [Q&A] [answered]
This was not an issue before upgrading to 2.5.0 (v2.5.0-1+np2+2.1)
executing `onedrive -s --resync` results in the following output and gets stuck at `Fetching items from the OneDrive API for the Drive ID:`

```
me@pcname:~$ onedrive -s --resync

D-Bus message bus daemon is available; GUI notifications are now enabled

The usage of --resync will delete your local 'onedrive' client state, thus no record of your current 'sync status' will exist.
This has the potential to overwrite local versions of files with perhaps older versions of documents downloaded from OneDrive, resulting in local data loss.
If in doubt, backup your local data before using --resync

Are you sure you wish to proceed with --resync? [Y/N] y

Deleting the saved application sync status ...
Using IPv4 and IPv6 (if configured) for all network operations
Attempting to contact Microsoft OneDrive Login Service
Successfully reached Microsoft OneDrive Login Service
Configuring Global Azure AD Endpoints
Fetching items from the OneDrive API for Drive ID: <myID>.............................................................................................................................................................................................................................................................................
```

I tried following the exact installation guide for Linux Mint 22, deleting anything that was left after using 2.4.x, i've tried connecting to different networks to see if it was bad network.

It seems to have been  some kind of access to my drive, since calling `onedrive -s --dry-run` before doing `onedrive -s` started listing all my files. but when it came to syncing, nothing happened. The initial dry run took so long to complete that i ended up CTRL-C-ing out of it.

I'm not the biggest network guy, so i'm completely out of my league when it comes to troubleshooting

**System info**
Onedrive version: 2.5.0-1+np2+2.1
OS: Linux Mint wilma 22 x86_64
Kernel: Linux 6.8.0-45-generic

if i left out some information, please let me know, thanks in advance

**Comment by abraunegg** [comment, marked as answer]
There is no issue here - those are processing dots indicating it is doing something.

Add `--verbose` and you will see more info.

Please read *all* the documentation

**Reply by Langtude-Larson** [reply]
You're completely right, it was so many minute's I thought it had frozen, but I just had to trust the process, thanks for taking the time to answer my silly question
